scrapers/nv/events.py

```diff
@@ -1,82 +1,167 @@
+            yield "12:00 am"
+            return
+        time, ampm = start.groups(1)
+        yield f"{time}:00 {ampm}"
+
+
+class Agenda(PdfPage):
+    def process_page(self):
+        # Simplify text to make regex simpler
+        self.text = self.text.replace(". ", "").replace(".", "")
+
+        # Find all bill ids
+        bills = bills_re.findall(self.text)
+
+        # Format bills before yielding
+        formatted_bills = set()
+        for alpha, num in bills:
+            formatted_bills.add(f"{alpha.upper()} {num}")
+
+        yield from formatted_bills
+
+
+class Meetings(HtmlPage):
+    source = "https://www.leg.state.nv.us/App/NELIS/REL/82nd2023/Meetings"
+
+    def process_page(self):
+        # Use session selector dropdown menu to find latest session
+        session_id = (
+            XPath('//div[@aria-labelledby="session"]/a[1]/@href')
+            .match_one(self.root)
+            .rsplit("/", 1)[1]
         )
 
-        match_coordinates(
-            event,
-            {
-                "555 e. washington ave": (36.18278967322847, -115.13231621695132),
-                "legislative building": (39.16196376710227, -119.76626916663172),
+        session_link = URL(
+            url=f"https://www.leg.state.nv.us/App/NELIS/REL/{session_id}/Meetings?Meetings=System.Collections.Generic.List%601%5BNvLeg.Models.Nelis.SessionManager.Views.MeetingComposite%5D&MeetingsFilters=System.DateTime%5B%5D&ShowFloorMeetings=True&ShowCommitteeMeetings=True&ShowConferenceCommittees=True",
+            method="POST",
+            data={
+                "MeetingsFilters": [
+                    "1/1/2000",
+                    "1/1/2999",
+                ],
+                "ShowCommitteeMeetings": [
+                    "true",
+                    "false",
+                ],
+                "ShowFloorMeetings": "false",
+                "ShowConferenceCommittees": "false",
+                "X-Requested-With": "XMLHttpRequest",
             },
         )
+        yield CurrentMeetings(source=session_link)
 
-        event.add_source(self.URL)
 
-        if "committee" in title.lower():
-            event.add_committee(title, note="host")
+class CurrentMeetings(HtmlPage):
+    _TZ = pytz.timezone("PST8PDT")
+
+    def process_page(self):
+        meetings = []
+        cur_row = []
+        # Each meeting has info spread across up to 3 divs, with a <hr> after
+        # each section - The following code finds all of these sections with 3
+        # divs and places them into the meetings variable
+        for i in self.root.getchildren():
+            if i.tag == "div":
+                cur_row.append(i)
+            elif i.tag == "hr":
+                # Only add if all three elements are present - missing elements
+                # indicate that this section is cancelled or floor meeting
+                if len(cur_row) == 3:
+                    meetings.append(cur_row)
+                cur_row = []
```

Comment:
Are you _sure_ you want to empty this higher scoped variable here?

Reply:
Yes - If there aren't exactly 3 elements in cur_row when a <hr> is found, then cur_row should be emptied without adding it to the meetings list.

Reply:
Seems reasonable. Can you add a quick note explaining why you're emptying the variable? I could see someone coming back to this later and being confused.